### PR TITLE
Add additional tests for transcript functionality

### PR DESCRIPTION
**Pull request by igorkravchenko**
Add extra tests for `Availability` that were unlocked with the new release of CoreSDK along with more tests for secure transcript model.

MOB-2078

### CODE REVIEW

File: GliaWidgetsTests/SecureConversations/AvailabilityTests.swift [hunk 1]
```diff
@@ -33,4 +33,71 @@ final class AvailabilityTests: XCTestCase {
...
+
+    func testAvailableQueueStatus() throws {
+        var env = Availability.Environment.failing
+        let queueId = UUID.mock.uuidString
+        env.listQueues = { callback in
+            callback([.mock(id: queueId, status: .open, media: [.messaging])], nil)
+        }
+        env.isAuthenticated = { true }
+        env.queueIds = [UUID.mock.uuidString]
+        let availability = Availability.init(environment: env)
```
Comment (gersonnoboa) [inline GliaWidgetsTests/SecureConversations/AvailabilityTests.swift:45]: Probably the `init` is not needed. There's several instances of this throughout the tests after this one.

Comment (igorkravchenko) [reply GliaWidgetsTests/SecureConversations/AvailabilityTests.swift]: Explicit `init` is only for autocompletion. cb49477b043b9f0cb3e4d08800ee7bb814bc664b

File: GliaWidgetsTests/SecureConversations/TranscriptModelTests.swift [hunk 2]
```diff
@@ -148,4 +149,233 @@ final class TranscriptModelTests: XCTestCase {
...
+            ),
+            deliveredStatusText: ""
+        )
+
+        viewModel.start()
+        XCTAssertEqual(calls, [.getSecureUnreadMessageCount])
+    }
+
+    func testLoadHistoryAddsUnreadMessageDivider() {
+        func testLoadHistoryAlsoInvokesUnreadMessageCount() {
```
[the remark this answers is not in the document]
Comment (igorkravchenko) [reply GliaWidgetsTests/SecureConversations/TranscriptModelTests.swift]: It is copy-paste. Also extra stuff had to be added in that test to pass af6d85b6c13db67fda4eb8bc59f4fc9fec12d71e.

**Review by yurii-glia — APPROVED** [review]

**Review by EgorovEI — APPROVED** [review]

**Review by gersonnoboa — APPROVED** [review]

**Comment by igorkravchenko** [comment]
!squash

**Comment by igorkravchenko** [comment]
!merge

**Pull request merged**